src/views/Settings.vue

```diff
@@ -38,6 +38,7 @@
 
       <section>
         <DxpOmsInstanceNavigator />
+        <DxpFacilitySwitcher @updateFacility="handleFacilityUpdate()"/>
```

Comment:
this is the wrong place, facility selector should be after the product store selector. @ravilodhi